server/controllers/inventory/depots/index.js

```diff
@@ -90,7 +105,7 @@ function update(req, res, next) {
         SELECT BUID(uuid) as uuid, text, enterprise_id, is_warehouse,
           allow_entry_purchase, allow_entry_donation, allow_entry_integration, allow_entry_transfer,
           allow_exit_debtor, allow_exit_service, allow_exit_transfer, allow_exit_loss,
-          min_months_security_stock
+          min_months_security_stock, IF(parent_uuid IS NULL, 0, BUID(parent_uuid)) as parent_uuid
```

Comment:
Just FYI - there is actually a function for this: [`IFNULL(x, y)`](https://www.w3schools.com/sql/func_mysql_ifnull.asp).